Ansible/deployvms.yml

```diff
@@ -78,6 +78,7 @@
       shell: "cmk addAccountToProject account={{ item }} projectid={{ projectid.stdout }}"
       with_items:
         - "{{ allaccts_retval.stdout.split(',') }}"
+      ignore_errors: true
```

Comment:
so this will render some trillian environment unusable to some users, by circumstance?

Reply:
right, but they can be granted access by an admin later on. Better to do that than fail here